openshift/ci/tests/e2e-ansible-scaffolding.sh

```
@@ -143,17 +143,23 @@ EOF
-echo "Configuring deployment memory limit to 1Gi for cluster-wide secret watching"
-cat <<EOF > config/manager/memory_patch.yaml
+# Deploy the operator: copies testdata to a scratch dir, applies the
+# pre-deploy patches (memory limit, project.openshift.io RBAC), runs
+# `make deploy`, creates the metrics clusterrolebinding, and switches to
+# the operator's namespace. Leaves the caller pushd'd into the scratch
+# copy so it can continue with test_operator/make undeploy.
+deploy_operator() {
+    echo "Creating temporary copy of test operator to avoid modifying synced upstream files"
+    TESTDIR="$(mktemp -d)"
+    trap_add "rm -rf $TESTDIR" EXIT
+    cp -r $ROOTDIR/testdata/memcached-molecule-operator $TESTDIR/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)